README.md

```diff
@@ -58,12 +58,24 @@ You can also submit a feature request [here](https://portal.productboard.com/zen
 
 ## Root Endpoint
 
-On Cloud, the root endpoint for the public API is `https://api.zenhub.com/`.
-For ZenHub Enterprise, the root endpoint is `https://<zenhub_enterprise_host>/`.
+The ZenHub API root endpoint for Cloud is different than that of Enterprise and has also changed across Enterprise versions. Please refer to the table below for the appropriate endpoint.
```

Comment:
Just want to make it super clear that this is ZenHub Enterprise, not GitHub Enterprise.

```suggestion
The ZenHub API root endpoint for Cloud is different than that of ZenHub On-Premise Enterprise instances and has also changed across Enterprise versions. Please refer to the table below for the appropriate endpoint.
```